earthpy/tests/test_plot.py

```diff
@@ -16,7 +15,7 @@
 
 # Create tuple
 tuups = (1, 2)
-im = np.random.randint(10, size=(2, 4, 5))
+im_arr = np.random.randint(10, size=(2, 4, 5))
 
 def test_arr_parameter():
```

Comment:
Looks like this test can be simplified by not using the global variable `tuups`, e.g., 

```python
def test_arr_parameter():
    """Raise an AttributeError if an array is not provided."""
    with pytest.raises(AttributeError):
        es.plot_bands(arr=(1, 2))
```

Reply:
yes!! that is what i decided to do through. thank you for the suggestion!!